docs/index.md

```diff
@@ -1,33 +1,54 @@
-# Why are we building OLM v1?
-
-Operator Lifecycle Manager's mission has been to manage the lifecycle of cluster extensions centrally and declaratively on Kubernetes clusters. Its purpose has always been to make installing, running, and updating functional extensions to the cluster easy, safe, and reproducible for cluster administrators and PaaS administrators, throughout the lifecycle of the underlying cluster. 
+# What Operator Lifecycle Manager (OLM)?
 
-OLM v0 was focused on providing unique support for these specific needs for a particular type of cluster extension, which have been coined as [operators](https://operatorhub.io/what-is-an-operator#:~:text=is%20an%20Operator-,What%20is%20an%20Operator%20after%20all%3F,or%20automation%20software%20like%20Ansible.). 
-Operators are classified as one or more Kubernetes controllers, shipping with one or more API extensions (CustomResourceDefinitions) to provide additional functionality to the cluster. After running OLM v0 in production clusters for a number of years, it became apparent that there's an appetite to deviate from this coupling of CRDs and controllers, to encompass the lifecycling of extensions that are not just operators.
+Operator Lifecycle Manager (OLM) is an open-source [CNCF](https://www.cncf.io/) project with the mission to manage the 
+lifecycle of cluster extensions centrally and declaratively on Kubernetes clusters. Its is to make installing, 
+running, and updating functional extensions to the cluster easy, safe, and reproducible for cluster administrators and PaaS administrators, 
+throughout the lifecycle of the underlying cluster.
 
-OLM has been helping to define lifecycles for these extensions in which the extensions
+OLM was focused on a particular type of cluster extension: [operators](https://operatorhub.io/what-is-an-operator#:~:text=is%20an%20Operator-,What%20is%20an%20Operator%20after%20all%3F,or%20automation%20software%20like%20Ansible.). 
+An operator is a Kubernetes application composed of one or more controllers and one, or more, API extensions, 
+or [CustomResourceDefinitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)) (CRDs).
 
-  * get installed, potentially causing other extensions to be installed as well as dependencies
-  * get customized with the help of customizable configuration at runtime 
-  * get upgraded to newer version/s following upgrade paths defined by extension developers 
-  * and finally, get decommissioned and removed.
+OLM helped define lifecycles for these extensions in which the extensions:
 
-In the dependency model, extensions can rely on each other for required services that are out of scope of the primary purpose of an extension, allowing each extension to focus on a specific purpose. 
+* get installed, potentially causing other dependency extensions to be installed as well
```

Comment:
I was under the impression that we are not doing any dependency resolution as part of OLM 1.0.

Reply:
I don't think it's our highest priority, but,  I also don't think it's been completely ruled out ^^ (I'm sure PM really wants it XD)